api/community/views.py

```diff
@@ -1,3 +1,54 @@
 from django.shortcuts import render
+from .models import Group, Topic, Hashtag, Media, Event
+from rest_framework import generics, permissions
+from .serializers import GroupSerializer, TopicSerializer, HashtagSerializer, MediaSerializer, EventSerializer
 
-# Create your views here.
+class GroupList(generics.ListCreateAPIView):
+    queryset = Group.objects.all()
+    serializer_class = GroupSerializer
+    permissions_classes = (permissions.IsAuthenticated)
+
+class GroupDetail(generics.RetrieveUpdateDestroyAPIView):
+    queryset = Group.objects.all()
+    serializer_class = GroupSerializer
+    permissions_classes = (permissions.IsAuthenticated)
```

Comment:
Do we want to have full crud on all these resources through the API? I think probably just read for List and Detail. Right?

Reply:
Oh maybe not - can users create groups?

Reply:
@ZakkMan We were thinking that we could allow full CRUD for community related data models. Our understanding is that this content is facilitated and managed by users, whereas trip data would be delegated to admins.